src/CodeViewerDemo/CodeExamples.h

```diff
@@ -226,19 +226,19 @@ class GrpcCaptureEventSender final : public CaptureEventSender {
 
 }  // namespace
 
-// LinuxTracingHandler::Stop is blocking, until all perf_event_open events have been processed
+// TracingHandler::Stop is blocking, until all perf_event_open events have been processed
 // and all perf_event_open file descriptors have been closed.
 // CaptureStartStopListener::OnCaptureStopRequested is also to be assumed blocking,
 // for example until all CaptureEvents from external producers have been received.
 // Hence why these methods need to be called in parallel on different threads.
 static void StopTracingHandlerAndCaptureStartStopListenersInParallel(
-    LinuxTracingHandler* tracing_handler,
+    TracingHandler* tracing_handler,
     absl::flat_hash_set<CaptureStartStopListener*>* capture_start_stop_listeners) {
   std::vector<std::thread> stop_threads;
 
   stop_threads.emplace_back([&tracing_handler] {
     tracing_handler->Stop();
-    LOG("LinuxTracingHandler stopped: perf_event_open tracing is done");
+    LOG("TracingHandler stopped: perf_event_open tracing is done");
```

Comment:
This is actually not code but a big string that was copied from some of our code to test syntax highlighting :D

Reply:
Yeah I know... doesn't hurt to keep that up to date as well :-)